$NetBSD$

elab_net.cc:1323:6: error: 'auto_ptr' was not declared in this scope
elab_net.cc:1323:22: error: expected primary-expression before '>' token
elab_net.cc:1323:64: error: 'etmp' was not declared in this scope

--- elab_net.cc~	2008-12-10 12:21:48.000000000 +0900
+++ elab_net.cc	2014-08-20 17:08:45.000000000 +0900
@@ -29,6 +29,7 @@
 
 # include  <iostream>
 # include  <cstring>
+# include  <memory>
 
 /*
  * This is a state flag that determines whether an elaborate_net must
